pkg/service/v0/service.go

```diff
@@ -143,6 +144,7 @@ func New(opts ...Option) (s *Service, err error) {
 	// TODO don't bother to store fields as we will load the account from disk
 
 	s = &Service{
+		id:     cfg.GRPC.Namespace + cfg.Server.Name,
```

Comment:
With our default values this will evaluate to `com.owncloud.apiaccounts`, missing a `.` between the namespace and the name, right?

Reply:
Nice catch. :+1: